commandapi-platforms/commandapi-bukkit/commandapi-bukkit-nms/commandapi-bukkit-1.21.4/src/main/java/dev/jorel/commandapi/nms/NMS_1_21_R3.java

```diff
@@ -240,6 +246,14 @@ public class NMS_1_21_R3 extends NMS_Common {
 		// For some reason, MethodHandles fails for this field, but Field works okay
 		serverFunctionLibraryDispatcher = CommandAPIHandler.getField(ServerFunctionLibrary.class, "h", "dispatcher");
 
+		MethodHandles.Lookup lookup = MethodHandles.lookup();
+		try {
+			minecraftServerSetSelected = lookup.findVirtual(PackRepository.class, "setSelected", MethodType.methodType(void.class, Collection.class, boolean.class));
+			setSelectedWithBoolean = true;
+		} catch (NoSuchMethodException | IllegalAccessException e) {
+			// So, uhh... I guess we're on Paper 1.21.4 build 62 or earlier
```

Comment:
Adjusting this comment does not matter at all :P, but technically we do expect to sometimes be on Spigot

```suggestion
			// We're on Spigot or Paper 1.21.4 build 62 or earlier
```